.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 dist
 .mypy_cache
+pyfxa_versions.egg-info
```

Comment:
When running `python setup.py sdist` or something along those lines, it generates this directory of packaging metadata. It's worth ignoring because you don't want to check it into the repo.

Reply:
:+1: